interpreter/cli/cli.py

```diff
@@ -1,10 +1,8 @@
 import argparse
-import subprocess
 import os
-import platform
 import pkg_resources
 import appdirs
-from ..utils.display_markdown_message import display_markdown_message
```

Comment:
Cleaning up imports that are no longer needed due to the `open_file` utility, and removing `display_markdown_message` as it is no longer used in this file.